ch-06/src/Solution618.java

```diff
@@ -0,0 +1,45 @@
+import java.util.Scanner;
+
+public class Solution618 {
+    public static void main(String[] args) {
+        final Scanner input = new Scanner(System.in);
+        final String password = input.nextLine();
+
+        passwordChecker(password);
+    }
+    public static boolean isPasswordLongEnough(String str) {

```

Comment:
Parameters should always be `final` (except for the `main` method, where we usually don't care, for historical reasons).